contributors/devel/help-wanted.md

```diff
@@ -2,12 +2,91 @@
-- Up to date: Often these issues become obsolete and have already been done, are no longer desirable, no longer make sense, change in priority, change in difficulty, etc.
+- **Low Barrier to Entry**<br/>
+  It should be tractable for casual contributors. Documentation on how that type of change should be made should already exist.
+
+- **Clear Task**<br/>
+  The task is agreed upon and does not require further discussions
+  in the community. Call out if that area of code is untested and requires new fixtures.
+
+  API / CLI behavior is decided and included in the OP issue, for example: _"The new command syntax is `svcat unbind NAME [--orphan] [--timeout 5m]`"_, with expected validations called out.
+
+- **Goldilocks priority**<br/>
+  Not too high that a core contributor should do it, but not too low that it
+  isn't useful enough for a core contributor to spend time to review it, answer questions, help get it into a release, etc.
+
+- **Up-To-Date**<br/>
+  Often these issues become obsolete and have already been done, are no longer desired,
+  no longer make sense, have changed priority or difficulty , etc.
 
 Related commands:
 
 - `/help` : adds the `help wanted` label to an issue
-- `/remove-help` : removes the `help wanted` label from an issue
\ No newline at end of file
+- `/remove-help` : removes the `help wanted` label from an issue
+
+# Good First Issue
+
+Items marked with the `good first issue` label are intended for _new contributors_.
+These make it clear to new contributors that they are welcome, valued, and provide
+a clear starting point for joining the project. After a contributor has successfully
+completed 1-2 `good first issue`'s, they should be ready to move on to `help wanted` items, saving remaining `good first issue`'s for other new contributors.
+
+Please keep an eye out for pull requests from new contributors and help
+move it through our processes. New contributors shouldn't be left to decipher prow
+commands, build flakes, find an approver, or ping for reviews.
+
+These items need to ensure that they follow the guidelines for `help wanted` labels (above)
+in addition to meeting the following criteria:
+
+- **No Barrier to Entry**<br/>
+  The task is something that a new contributor can tackle without
+  advanced setup, or domain knowledge.
+
+- **Solution Explained**<br/>
+  The recommended solution is clearly described in the issue.
+
+- **Provides Context**<br/>
+  If background knowledge is required, this should be explicitly mentioned
+  and a list of suggested readings included.
+
+- **Gives Examples**<br/>
+  Link to examples of similar implementations so new contributors have a reference guide
+  for their changes.
+
+- **Identifies Relevant Code**<br/>
+  The relevant code and tests should be linked in the issue.
```

Comment:
nit: relevant code _to be changed_?

This could also probably mean code that _relates_ to the issue at hand but not the exact location of the code where the change needs to occur. I have usually seen that most new contributors face a lot of trouble in identifying the file they need to modify. :)